visualisation_of_niches/NicheCompass_paper_sunburst_plot.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>Why do you extract gene weights for 20 genes but in the previous cell, you extracted 60 genes? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Talavera-Lopez-Lab/Fetal_stem_cells_analysis/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Because I forgot to change the cell 🙈</p>

Reply:
Solved now, please check update.